### DISCUSSION TITLE
Tasmota Neopool Switch Aux On/Off

### DISCUSSION THREAD

**Discussion by Vippis2000** [arendst/Tasmota#21219] [Support request]
Hi, I have managed to setup Neopool for my Brilix Hidrolife system.

I want to connect a second ph-Pump to the Aux2 Relay, which will pump base liquid (ph+ liquid) into the pool for pH-regulation in case the pH-value is below pH-min-value, causing the ph-value to increase.

How can I start/stop the pump using NP-Commands?
@curzon01 @alexdelprete

Best regards

**Comment by curzon01** [comment]
Firstly, there is an automation within Brilix for pH+ that makes the manual switching of Aux2 unnecessary. You have to configure your Brilix so that it is an Acid+Base system, then Aux2 is configured as a pH+ pump and the system controls both pumps (ph-/ph+) based on the min/max pH values and the control settings (but this is not a Tasmota topic).

Convenient manual switching of the Aux relays is possible using the optional `NPAux<x>` command.
Requirement: An ESP32 and the [Berry command extension for NeoPool](https://tasmota.github.io/docs/NeoPool/#esp32-adding-user-defined-neopool-commands-to-tasmota).

**Comment by curzon01** [comment]
> I want to connect a second ph-Pump to the Aux2 Relay, which will pump base liquid (ph+ liquid) into the pool for pH-regulation in case the pH-value is below pH-min-value, causing the ph-value to increase.

In general: With an electrolysis/hydrolysis system, a ph+ pump is not actually necessary. The pH value increases automatically through electrolysis/hydrolysis. The ph+ option is actually only necessary for systems that automate disinfection using liquid chlorine. There, the pH value can also fall below a minimum value, e.g. by adding fresh water when the water is acidic.

**Comment by Vippis2000** [comment]
Thank you for your explanation. I didn't know that in a hydrolisis system pH rises automatically.

**Reply by curzon01** [reply]
> I didn't know that in a hydrolisis system pH rises automatically.

To test, just leave the pH minus pump off, you will notice that the pH value increases due to the hydrolysis. This is why a pH- pump is always recommended for such systems.
Unfortunately, the pH- liquid (usually sulphuric or hydrochloric acid) lowers the alkalinity. This value must also be monitored and should not fall below 30-20 as otherwise the pH value may no longer be kept stable. Normally this is achieved by a sufficient supply of fresh water (after backwashing), if not, you can increase the alkalinity with baking soda (sodium bicarbonate), but this also increases the pH value again - a vicious circle.

**Comment by alexdelprete** [comment]
> Thank you for your explanation. I didn't know that in a hydrolisis system pH rises automatically.

I highly suggest reading the articles in this very useful site, and they also have a great community: https://www.troublefreepool.com/blog/pool-school/
